init-ci.gradle

```diff
@@ -0,0 +1,22 @@
+settingsEvaluated {
+    it.pluginManagement {
+        repositories {
+            maven {
+                url System.getenv("ARTIFACTORY_URL") ?: 'https://artifacts.openmicroscopy.org/artifactory/maven/'
+            }
+            maven {
+                url System.getenv("GITLAB_URL") ?: 'https://artifacts.openmicroscopy.org/artifactory/maven/'
+            }
+            maven {
+                url System.getenv("MAVEN_RELEASES_REPO_URL") ?: 'https://artifacts.openmicroscopy.org/artifactory/maven/'
+            }
+            maven {
+                url System.getenv("MAVEN_SNAPSHOTS_REPO_URL") ?: 'https://artifacts.openmicroscopy.org/artifactory/maven/'
+            }
+            gradlePluginPortal()
+        }
+    }
+}
+
+// Use a local m2 repo to ensure a clean build
+System.setProperty("maven.repo.local", System.getProperty("user.dir") + "/m2/repository")
```

Comment:
not `.m2`?

Reply:
No, this is a custom location, no need to hide it with a `.`

Reply:
FYI `user.dir` refers to the initial working/build directory